src/chat/infinite.js

```diff
@@ -787,6 +787,16 @@ export async function infiniteMessageArea (dom, kb, chatChannel, options) {
       !earliest.messageTable.initial &&
       earliest.messageTable.extendBackwards
     ) {
+      // If this has been called before the element is actually in the
+      // user's DOM tree, then this scrollTop check won't work -> loop forever
```

Comment:
**question:** I don't get it from the comment yet: Is the "loop forever" fixed by these lines, or is there still an issue left?

Reply:
Its fixed by spotting that scrolHeight is zero, and putting off the load of the chat till 2 secs later